files/default/handler/audit_report.rb

```diff
@@ -105,18 +105,35 @@ def get_opts(reporter, quiet)
       # run profiles and return report
       def call(opts, profiles)
         Chef::Log.info "Initialize InSpec #{::Inspec::VERSION}"
-
         Chef::Log.debug "Options are set to: #{opts}"
         runner = ::Inspec::Runner.new(opts)
 
         # parse profile hashes for runner, see libraries/helper.rb
         tests = tests_for_runner(profiles)
-        tests.each { |target| runner.add_target(target, opts) }
+        if !tests.empty?
+          tests.each { |target| runner.add_target(target, opts) }
+
+          Chef::Log.info "Running tests from: #{tests.inspect}"
+          runner.run
+          r = runner.report
+
+          # output summary of InSpec Report in Chef Logs
+          unless r.nil? && ['json-min', 'json'].include(opts['format'])
+            time = 0
+            time = r[:statistics][:duration] unless r[:statistics].nil?
+            passed_controls = r[:controls].select { |c| c[:status] == 'passed' }.size
```

Comment:
Ah, I was running into the issue that controls are available in full reports, too. See https://github.com/chef/inspec/pull/1564. Since this is deprecated, we need to adapt this implementation.